src/repository/index.ts

```diff
@@ -13,6 +13,8 @@ import {
   Segment,
   StrategyTransportInterface,
 } from '../strategy/strategy';
+// @ts-expect-error
```

Comment:
this library does not provide any types so we have to do this for now until we provide a type ourselves